x-pack/plugins/cross_cluster_replication/server/routes/api/ccr.js

```diff
@@ -64,8 +64,8 @@ export const registerCcrRoutes = (server) => {
       }
 
       const securityInfo = (xpackInfo && xpackInfo.isAvailable() && xpackInfo.feature('security'));
-      if (!securityInfo || !securityInfo.isEnabled()) {
-        // If security isn't enabled, let the user use CCR.
+      if (!securityInfo || !securityInfo.isAvailable() || !securityInfo.isEnabled()) {
```

Comment:
@cjcenizal I discovered that this branch isn't entered if security is enabled, but not available. This can happen if user has a license with security enabled, and reverts back to Basic. Basic doesn't have security available, but security is still "enabled".

(I started going down the path of copying this logic over to Snapshots, but realized Snapshots is Basic so there isn't a need to check license info 🙂)

Reply:
Great catch! And thanks for adding the comment. I'm sure I'll completely forget about this and that comment will save the day for me.